app/main.py

```diff
@@ -13,7 +13,7 @@ def __init__(
         self.hidden = hidden
         Animal.alive.append(self)
 
-    def keep_alive(self) -> None:
+    def remove_expired_animals(self) -> None:
```

Comment:
`remove_dead_animal`
I don't think there's some expiration date here)
or, maybe something like this, for total explicity:
`remove_from_alive_if_dead`